2_base_and_sensor.ipynb

```diff
@@ -107,11 +205,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "HSRの台車は全方向に移動可能です。以下のように速度司令を入力し、左方向に移動させてみましょう。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The HSR can move in all directions. Enter the speed command as below and move to the left:"
```

Comment:
"The HSR can move in all directions. Enter the speed command as shown below and move to the left:"

If there are multiple commands, then:

"The HSR can move in all directions. Enter the speed commands as shown below and move to the left:"